ui/src/views/Quickstart/index.jsx

```diff
@@ -380,7 +380,7 @@ export default class QuickStart extends Component {
         <Fragment>
           <div className={classes.orgRepo}>
             <TextField
-              label="Organization Name"
+              label="Orga Name"
```

Comment:
I think GitHub uses the short name "Org".

Reply:
Made the change

Reply:
Please review 😄